packages/smooth_app/pubspec.lock

```diff
@@ -357,13 +357,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.2"
-  google_ml_barcode_scanner:
+  google_ml_kit:
```

Comment:
Ohh yes right @teolemon the changes in this file should be reverted

Reply:
I've made the required changes.